test_data/yast/raid1_lvm/lvm_raid1.yaml

```diff
@@ -0,0 +1,12 @@
+---
+arch: x86_64
+num_disks: 4
```

Comment:
Replace `num_disks` by this as I missed that we are using the names in a loop below:
```
disks:
  vda:
  vdb:
  vdc:
  vdd:
```

Reply:
done